packages/cycle-scripts/template/src/JavaScript/app.js

```diff
@@ -0,0 +1,12 @@
+module.exports = replacements => `${replacements.import}
+
+export function App (sources) {
```

Comment:
I would use the counter example as the base app, like I did [here](https://github.com/cyclejs-community/create-cycle-app-flavors/blob/master/packages/cycle-scripts-one-fits-all/template/src/app.tsx). Has the advantage of showing MVI